src/stores/AppStore.js

```diff
@@ -727,15 +726,20 @@ export default types
     }
 
     function setHistory(history = []) {
+      console.log('Setting annotation history');
       const as = self.annotationStore;
 
+      const selected = as.selected?.toJSON();
+
+      console.log({ pk: selected?.pk, id: selected?.id });
```

Comment:
are we looking to keep these console.logs temporarily? or are is this debug logs that were missed?
```suggestion
      const as = self.annotationStore;
      const selected = as.selected?.toJSON();
```